### ISSUE TITLE
Transaction does not contain data about TransactionError

### ISSUE THREAD

**Issue by PashaSk** [recurly/recurly-client-net#371]
I have a transaction that was declined due to fraud_risk_check. I can see <transaction_error> node inside XML output of an API call https://<env>.recurly.com/v2/transactions/:uid but Transaction.ReadXml method does not parse this fields. 
Have you any plans to implement this or is there any way to get this error_code with error messages? 
 

`<transaction_error>
<error_code>fraud_risk_check</error_code>
<error_category>fraud</error_category>
<merchant_message>Fraudulent Transaction: Recurly declined this transaction based on your fraud management settings. Please see the "Fraud Details" section on the transaction details page for additional information.</merchant_message>
<customer_message>This transaction was declined because it appears to be a fraudulent attempt. Please try a different card.</customer_message>
<gateway_error_code nil="nil" />
 </transaction_error>`

**Comment by aaron-suarez** [comment]
@PashaSk Thank you for bringing this to our attention. I'm not sure if this fixed the problem or not, but recently I fixed a bug where the TransactionError is not included in the `ToString` of a Recurly Exception. Check out the diff here:

https://github.com/recurly/recurly-client-net/pull/368/files#diff-62a718478cb7ddabdd857f41b636a8a1R54

This is not released yet, but it's scheduled to go out in version 1.15.0. Do you believe that this would fix your issue?

I'll take a closer look at this if not.

**Comment by PashaSk** [comment]
@aaron-suarez  thanks for your reply!
I guess this won't help me. Let me rephrase the question. 
Once I get a Transaction model by calling:
`var transaction = Transactions.Get(<some uid>);`
I get the model Transaction which does not contain specific information about "fraud_risk_check". I can see necessary for me fields in API call output, but Transaction.ReadXml method won't parse them to any model properties.
Hope explanation will help!

**Comment by aaron-suarez** [comment]
Ah, yes that does help. I will go ahead and look further into this and see about finding a solution. If you have a solution in mind, feel free to open a PR and I'd be happy to review and merge it. If not, I'll open one when I find a solution. 😄

**Comment by PashaSk** [comment]
@aaron-suarez  please review my PR https://github.com/recurly/recurly-client-net/pull/372

**Comment by aaron-suarez** [comment]
Closed via #372
